tuf-spec.md

```diff
@@ -979,9 +979,29 @@ repo](https://github.com/theupdateframework/specification/issues).
          "meta" : METAFILES
        }
 
-   METAFILES is the same is described for the snapshot.json file.  In the case
-   of the timestamp.json file, this MUST only include a description of the
-   snapshot.json file.
+   METAFILES is an object whose format is the following:
+
+       { METAPATH : {
+             "version" : VERSION,
+             ("length" : LENGTH, |
```

Comment:
```suggestion
             "length" : LENGTH, |
```

Reply:
Is this different from other "or"-groups in the spec, where we do use parentheses (see [#L734-L735](https://github.com/theupdateframework/specification/blame/97312ddd239746a213694ce772e9bccbf4123f39/tuf-spec.md#L734-L735) and [#L840-L841](https://github.com/theupdateframework/specification/blame/97312ddd239746a213694ce772e9bccbf4123f39/tuf-spec.md#L840-L841))?

If not, I suggest we use a consistent notation, with or without parentheses.

Reply:
Oh, I understand now. The lack of parentheses indicates that one of the values must be provided.

Reply:
But if only one of them is mandatory, then the routine from the client workflow that @joshuagl quotes above , i.e. 
> "up to the number of bytes specified in the timestamp metadata file"

would fail in the case timestamp only lists  `hashes`.

Reply:
Very good point. If we keep the METAPATH object as is with optional fields, then the client should fetch either `length` or an application specified size, and in the latter case compare to the `hash`.

Perhaps `length` should be mandatory here, the client workflow seems to imply that it is.